src/shipit_static_analysis/shipit_static_analysis/config.yml

```diff
@@ -2,19 +2,36 @@ target: obj-x86_64-pc-linux-gnu
 
 # All the clang checks used by the bot
 clang_checkers:
- - -*
- - clang-analyzer-deadcode.DeadStores
- - modernize-loop-convert
- - modernize-raw-string-literal
 # modernize-use-auto (controversial, see bug 1371052)
 # modernize-use-bool-literals (too noisy because of `while (0)` in many macros)
- - modernize-use-equals-default
- - modernize-use-equals-delete
- - modernize-use-nullptr
- - modernize-use-override
- - mozilla-*
- - performance-faster-string-find
- - performance-for-range-copy
- - readability-else-after-return
- - readability-misleading-indentation
- - readability-redundant-control-flow
+ - name: -*
+   publish: no
```

Comment:
Have you verifies that this `yml` syntax works as expected? I ask because I recently lost a lot of time debugging my CircleCI `yml` file, which required a double-indent after a `-`:

```yml
  - firefox-git:
      requires: ubuntu-dev # note this double-indented line (with 4 spaces)
```

But maybe that's just a bug in CircleCI's `yml` parser or validator.